packages/components/pages/f-offers/stories/Offers.stories.js

```diff
@@ -60,7 +60,7 @@ export const VOffersComponent = (args, { argTypes }) => ({
     }),
 
     watch: {
-        offersState: function (val) {
+        offersState: function (val) { // eslint-disable-line func-names
```

Comment:
```suggestion
        offersState (val) {
```

Does the function shorthand work here?

Reply:
I was a bit nervous changing code too much (mainly as I don't want to break anything across the repo!).

I'll flag this and try it in a separate PR if that sounds ok?

Reply:
Sure, sounds good to me.